ambry-rest/src/main/java/com.github.ambry.rest/NettyServer.java

```diff
@@ -62,6 +62,7 @@ public NettyServer(NettyConfig nettyConfig, NettyMetrics nettyMetrics,
     this.nettyMetrics = nettyMetrics;
     this.channelInitializer = channelInitializer;
     NettyRequest.bufferWatermark = nettyConfig.nettyServerRequestBufferWatermark;
+    NettyRequest.maxBytesPerRead = nettyConfig.nettyServerMaxBytesPerRead;
```

Comment:
So basically in 4.1.4 you cannot set the number of msgs to read per cycle, but in turn can set the max bytes to read per cycle. That means, you can read only one msg in one cycle, but you can choose how big the msg is.

I think we can just set this to some preconfigured number. Users may not care about this at all.